crowd-server/crowd_server/apps/image_label/models.py

```diff
@@ -1,3 +1,41 @@
 from django.db import models
+from django.contrib.auth.models import User
 
-# Create your models here.
+
+class ImageCategory(models.Model):
+    name = models.CharField(max_length=120)
+    question = models.CharField(max_length=300)
+    image_path = models.CharField(max_length=300)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    def __str__(self):
+        return f"{self.name} - {self.question}"
+
+
+class Image(models.Model):
+    image_category = models.ForeignKey(ImageCategory, on_delete=models.CASCADE, related_name='images')
+    path = models.CharField(max_length=300)
```

Comment:
If you are using an image, use `ImageField()`. Because it has validations for image files. And you can access its url easily from it.